README.md

```diff
@@ -1,90 +1,141 @@
+
+### Local development environment details
+
 This will run the following components:
 
-- Nginx, a web server: http://localhost:8000
-- Adminer, a DB front-end: http://localhost:8001
+- Nginx, a web server: http://localhost:8000 - view the application or make API calls
 - Mailhog, a dummy mailbox: http://localhost:8025
-- A PostgreSQL database
 - The Django web application
+- Adminer, a DB front-end: http://localhost:8001
+- A PostgreSQL database `postgres://babyyoda:mysecretpassword@localhost:5432/codebuddies`
 
-You can view the application or make API calls by using the Nginx URL.
+You can access the database through the Adminer front-end or using a local PostgreSQL client
 
-You can access the database through the Adminer front-end or using a local PostgreSQL client and the following URL: `postgres://babyyoda:mysecretpassword@localhost:5432/codebuddies`.
+![screenshot of Adminer](https://i.imgur.com/Dtg5Yel.png =250x)
 
-![screenshot of Adminer](https://i.imgur.com/Dtg5Yel.png)
+5. Create a superuser so that you can log into `http://localhost:8000/admin` by running the following in your terminal: 
 
-To stop the application and remove all containers, run the following.
+```bash
+$ docker-compose run --rm app ./manage.py createsuperuser
+```
 
-```plain
-docker-compose down
+6. You can populate the database with some random test data for development purposes by running 
+
+```bash
+$ docker-compose run --rm app ./manage.py init_data
+```
+
+All user accounts created by this command have the password `codebuddies`.
+
+See the `init_data --help` command for more information:
+
+```bash
+$ docker-compose run --rm app ./manage.py init_data --help
+
+usage: manage.py init_data [-h] [--clear-db] [--num-users NUM-USERS]
+                           [--num-tags NUM-TAGS]
+                           [--num-resources NUM-RESOURCES] [--version]
+                           [-v {0,1,2,3}] [--settings SETTINGS]
+                           [--pythonpath PYTHONPATH] [--traceback]
+                           [--no-color] [--force-color]
+
+Initialize the DB with some random fake data for testing and development
+
+optional arguments:
+  --clear-db            Clear existing data from the DB before creating test
+                        data
+  --num-users NUM-USERS
+                        Number of `User` objects to create (default 10)
+  --num-tags NUM-TAGS   Number of `Tag` objects to create (default 10)
+  --num-resources NUM-RESOURCES
+                        Number of `Resource` objects to create (default 10)
+  -v {0,1,2,3}, --verbosity {0,1,2,3}
+                        Verbosity level; 0=minimal output, 1=normal output,
+                        2=verbose output, 3=very verbose output
 ```
 
-5. Create a superuser so that you can log into http://localhost:8000/admin by running the following in your terminal: `$ docker-compose run --rm app ./manage.py createsuperuser`
+[See PR 127]
```

Comment:
I think this is supposed to say PR 129 so it can point to https://github.com/codebuddies/backend/issues/129

Reply:
How does this relate to the custom ./manage.py script and its random test data?

Reply:
`docker-compose run --rm app ./manage.py init_data` creates test data for things like users and resources, while `docker-compose run --rm app ./manage.py createsuperuser` creates a superuser that lets you log in to `/admin` and control the various data on the app. 

(Let me know if I answered your question/if that's what you were asking about!)